lib/providers/facebook.js

```diff
@@ -20,7 +20,8 @@ exports = module.exports = function (options) {
         profile: function (credentials, params, get, callback) {
 
             var query = {
-                appsecret_proof: Crypto.createHmac('sha256', this.clientSecret).update(credentials.token).digest('hex')
+                appsecret_proof: Crypto.createHmac('sha256', this.clientSecret).update(credentials.token).digest('hex'),
+                fields: 'id,name,email,first_name,last_name,middle_name'
```

Comment:
The graph query seams to return the fields only if they are explicitly provided.